src/main/java/seedu/anilist/logic/commands/RenameCommand.java

```diff
@@ -0,0 +1,152 @@
+package seedu.anilist.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.anilist.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.anilist.model.Model.PREDICATE_SHOW_ALL_ANIME;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+
+import seedu.anilist.commons.core.Messages;
+import seedu.anilist.commons.core.index.Index;
+import seedu.anilist.logic.commands.exceptions.CommandException;
+import seedu.anilist.model.Model;
+import seedu.anilist.model.anime.Anime;
+import seedu.anilist.model.anime.Episode;
+import seedu.anilist.model.anime.Name;
+import seedu.anilist.model.tag.Tag;
+
+public class RenameCommand extends Command {
+    public static final String COMMAND_WORD = "rename";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Rename the anime identified "
+        + "by the index number used in the displayed anime list. "
+        + "Existing values will be overwritten by the input values.\n"
+        + "Parameters: INDEX (must be a positive integer) "
+        + PREFIX_NAME + "NAME\n"
+        + "Example: " + COMMAND_WORD + " 1 " + PREFIX_NAME + "Hyouka";
+
+    public static final String MESSAGE_RENAME_ANIME_SUCCESS = "Edited Anime's Name: %1$s";
```

Comment:
should this be `Edited Anime name` to match the format of other commands' success messages?